src/templates/static/legal/tac_sections/affiliate.jsx

```diff
@@ -1,158 +1,358 @@
+                    },
+                    {
+                        text: it.L(
+                            'Commissions will be earned throughout the life of the client, on all transactions the client undertakes with the Company, for as long as the Affiliate remains a member of this affiliate programme. The Company reserves the right to cancel the commissions that are not reclaimed after 2 years.'),
+                    },
+                    {
+                        text: it.L(
+                            'The Affiliate is not allowed to rebate any part of the commission payment onto the Affiliate\'s referrals, and should the Company realise that the Affiliate is engaged in such actions, the Affiliate\'s account shall be revoked immediately.'),
+                    },
                 ]}
             />
 
             <ListNested
                 header={it.L('Warranties')}
                 items={[
-                    { text: it.L('Each party to this Agreement represents and warrants to the other that it has, and will retain throughout the term of this Agreement, all right, title, and authority to enter into this Agreement, to grant to the other party the rights and licences granted in this Agreement, and to perform all of its obligations under this Agreement.') },
-                    { text: it.L('The Affiliate represents, warrants, and undertakes that the Affiliate has obtained and shall maintain all necessary licences and consents to operate within any laws, rules, and regulations applicable under the authority of any regulatory body of the jurisdiction(s) in which the Affiliate operates or targets business.') },
-                    { text: it.L('The Affiliate represents, warrants, and undertakes that the Affiliate\'s website or promotion shall contain no material that is defamatory, pornographic, unlawful, harmful, threatening, obscene, harassing, or racially, ethnically, or otherwise objectionable or discriminatory, violent, politically sensitive, or otherwise controversial or in breach of any third party rights and shall not link to any such material.') },
+                    {
+                        text: it.L(
+                            'Each party to this Agreement represents and warrants to the other that it has, and will retain throughout the term of this Agreement, all right, title, and authority to enter into this Agreement, to grant to the other party the rights and licences granted in this Agreement, and to perform all of its obligations under this Agreement.'),
```

Comment:
honestly this isn't any more readable than how it used to be, if you look at the PR diff. can we revert these changes so it matches the rest of the files instead?

Reply:
Yes.